test_tools.py

```
@@ -0,0 +1,85 @@
+def test_tools():
+    """Test the MCP server tools."""
+    
+    # Start the MCP server
+    print("Starting MCP server...")
+    process = subprocess.Popen(
+        [sys.executable, "mcp-wrapper/mcp_server.py"],
+        stdin=subprocess.PIPE,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        text=True
+    )
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)